src/contract.rs

```diff
@@ -82,11 +72,6 @@ pub fn get_diamond_proxy_address(network: String) -> Address {
     } else if network == "testnet" {
         Address::from_str("1908e2bf4a88f91e4ef0dc72f02b8ea36bea2319").unwrap()
     } else {
-        panic!(
-            "Please use network name `{}`, `{}`, or `{}`",
-            "mainnet".yellow(),
-            "sepolia".yellow(),
-            "testnet".yellow()
-        );
+        Address::default()
```

Comment:
does it work? maybe it's better to return a Result<Address> in this method (or Option<Address>?)

Reply:
yea, by the time this gets called we've already validated the network being used. i can use a panic instead as the cli shouldnt work with an unknown network